include/libjungle/db_config.h

```diff
@@ -311,6 +313,16 @@ class DBConfig {
      * Default: `false`
      */
     bool directIo;
+
+    /**
+     * The size of memory buffer for direct-IO.
+     */
+    size_t directIoBufferSize;
+
+    /**
+     * The alignment size of memory buffer for direct-IO.
+     */
+    size_t directIoAlignSize;
```

Comment:
Since we will have more options for direct IO in the future, please put above 3 configs (including `bool directIo`) into a sub-structure, the same as `GlobalConfig::IdleTimeCompactionOptions` below. 
```C++
struct DirectIoOptions {
    ...
};

/**
 * Direct-IO related options.
 */
DirectIoOptions directIoOpt;
```